mara_metabase/metadata.py

```diff
@@ -13,10 +13,15 @@
 def update_metadata() -> bool:
     """Updates descriptions of tables & fields in Metabase, creates metrics and flushes field caches"""
     client = MetabaseClient()
-
-    dwh_db_id = next(filter(lambda db: db['name'] == config.metabase_data_db_name(),
-                            client.get('/api/database/')),
-                     {}).get('id')
+    all_dbs = client.get('/api/database/')
+
+    # metabase changed the API response sometimes between 38 and 41...
+    if isinstance(all_dbs, dict):
+        all_dbs = all_dbs["data"]
+    dwh_db_id = None
+    for db in all_dbs:
+        if db['name'] == config.metabase_data_db_name():
+            dwh_db_id = db['id']
```

Comment:
shouldn't we use a `break` here after we found the right db in the `all_dbs` list?

Reply:
yes... done